modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/upgrade/UpgradeCatchAllCheck.testjava

```diff
@@ -386,6 +386,10 @@ public class UpgradeCatchAllCheck {
 		_addressService.addAddress(userId, className, classPK, street1, street2, street3, city, zip,regionId, countryId, typeId, mailing, primary, serviceContext);
 	}
 
+	public void method(PortletDisplay portletDisplay) {
+		portletDisplay.getPortletInstanceConfiguration(AssetPublisherPortletInstanceConfiguration.class);
+	}
+
```

Comment:
@MicaelleSilvaa, the deleted Java test files cover 2 possibilities of code changes and here there is just one, can you please add here the 2 possibilities?

Thanks